project_script/ramp_time_treatment.py

```diff
@@ -83,20 +65,23 @@ def integrate_data():
 
 sanitize_data_frames()
 
-data_ramp_time = pd.concat([time_photo_celiv_ramp_time, odd_columns_subtracted], axis=1)\
-    .sort_index(axis=1)
+data_ramp_time = pd.concat([time_photo_celiv_ramp_time, odd_columns_subtracted], axis=1).sort_index(axis=1)
 
 # data_ramp_time.to_excel("C:/Users/robee/Desktop/data_ramp_time.xlsx")
 
 # data_ramp_time.to_csv(ask.file_name(), sep='\t')     # line commented to run tests
 
 data_ramp_time_transposed_in_arrays = data_ramp_time.transpose().to_numpy()
 
-time_photo_celiv_ramp_time_transposed = time_photo_celiv_ramp_time.dropna().transpose().to_numpy()
+time_photo_celiv_ramp_time_transposed = time_photo_celiv_ramp_time.dropna()\
+    .transpose()\
+    .to_numpy()
 
 odd_columns_subtracted_transposed_in_arrays = odd_columns_subtracted.transpose().to_numpy()
 
-integration_results = integrate_data()
+integration_results = dt.integrate_data(
+    odd_columns_subtracted_transposed_in_arrays, time_photo_celiv_ramp_time_transposed
```

Comment:
This line should be one more space forward, 'cause the code it's inside of some parameters, like this:

![image](https://user-images.githubusercontent.com/11571434/159492348-e264373a-8c56-425f-aa5a-91fa05d65b98.png)